crates/rnote-engine/src/engine/rendering.rs

```diff
@@ -35,21 +34,20 @@ impl Engine {
                     }
                 };
 
-                for split_bounds in viewport.split_extended_origin_aligned(
+                let origin_aabb = viewport.get_origin(
                     self.document.config.background.tile_size(),
                     SplitOrder::default(),
```

Comment:
What's the split order used for here? I don't understand why it is an argument in the first place.